packages/eslint-plugin/docs/rules/return-await.mdx

```diff
@@ -214,6 +218,35 @@ async function validAlways3() {
 </TabItem>
 </Tabs>
 
+### `error-handling-correctness-only`
+
+Same as `in-try-catch`, except _only_ enforces Promises be awaited within the
+error handling contexts explained under that option. Outside of error-handling
+contexts, returned Promises are ignored, whether awaited or not.
```

Comment:
Don't wrap lines by line width

Reply:
Huh. not sure how that happened. In any case this has been fixed/superseded 👍